tests/unit/uncertainty/test_estimators.py

```diff
@@ -120,6 +121,18 @@ def test_EvidentialEstimators(
     torch.testing.assert_close(uncs, expected_uncs)
 
 
+def test_QuantileRegressionEstimator(data_dir, dataloader, trainer):
+    model = MPNN.load_from_file(data_dir / "example_model_v2_regression_quantile_mol.pt")
+    estimator = QuantileRegressionEstimator()
+    preds, uncs = estimator(dataloader, [model], trainer)
+
+    print("preds", preds.tolist())
+    print("uncs", uncs.tolist())
+
+    torch.testing.assert_close(preds, torch.tensor([[[2.183332], [2.2001247]]]))
+    torch.testing.assert_close(uncs, torch.tensor([[[0.29111385], [0.3591898]]]))
```

Comment:
Using `@pytest.mark.parametrize`?

Reply:
I can't figure out how to parametrize this test. The checkpoint file fixes the outputs, and `QuantileRegressionEstimator`'s constructor has no arguments.

Reply:
That is a good point. I think let's keep what it is.